### PR TITLE
Support for updating by specified key field

### PR DESCRIPTION
**Pull request by ryokdy**
updateKey has only it's field code and value so that I had to separate marshaling fields as JSONValue interface.
https://cybozudev.zendesk.com/hc/ja/articles/201941784

### CODE REVIEW

File: app.go [hunk 1]
```diff
@@ -511,6 +525,40 @@ func (app *App) UpdateRecord(rec *Record, ignoreRevision bool) error {
 	return err
 }
 
+// UpdateRecord edits a record by specified key field.
```
Comment (ymmt2005) [inline app.go]: `// UpdateRecordByKey ...`

File: app.go [hunk 2]
```diff
@@ -550,6 +598,50 @@ func (app *App) UpdateRecords(recs []*Record, ignoreRevision bool) error {
 	return err
 }
 
+// UpdateRecords edits multiple records by specified key fields at once.
```
Comment (ymmt2005) [inline app.go]: `// UpdateRecordsByKey ...`

**Comment by ryokdy** [comment]
Thank you for your reviewing. I revised comments.

**Comment by ymmt2005** [comment]
LGTM

**Pull request merged**